convert_hf_to_gguf.py

```diff
@@ -669,7 +671,8 @@ def get_vocab_base_pre(self, tokenizer) -> str:
         if chkhsh == "7967bfa498ade6b757b064f31e964dddbb80f8f9a4d68d4ba7998fcf281c531a":
             # ref: https://huggingface.co/jinaai/jina-embeddings-v2-base-code
             res = "jina-v2-code"
-        if chkhsh == "b6e8e1518dc4305be2fe39c313ed643381c4da5db34a98f6a04c093f8afbe99b" or chkhsh == "81d72c7348a9f0ebe86f23298d37debe0a5e71149e29bd283904c02262b27516":
+        if chkhsh == "b6e8e1518dc4305be2fe39c313ed643381c4da5db34a98f6a04c093f8afbe99b" or chkhsh == "81d72c7348a9f0ebe86f23298d37debe0a5e71149e29bd283904c02262b27516" or chkhsh == "a1336059768a55c99a734006ffb02203cd450fed003e9a71886c88acf24fdbc2":
+            # ref: https://huggingface.co/THUDM/glm-4-9b-hf
```

Comment:
Did you make this change with `convert_hf_to_gguf_update.py`? There is a comment block above:

```
    # NOTE: this function is generated by convert_hf_to_gguf_update.py
    #       do not modify it manually!
    # ref:  https://github.com/ggml-org/llama.cpp/pull/6920
    # Marker: Start get_vocab_base_pre
```

I've never done ran that myself so I don't know how to use it but the style here looks inconsistent with other models, so makes me think it wasn't used?

Reply:
I did not make any changes; it seems the AI IDE wrote it.

Reply:
> Did you make this change with `convert_hf_to_gguf_update.py`? There is a comment block above:您是否用 `convert_hf_to_gguf_update.py` 做了这个更改？上面有一个注释块：
> 
> ```
>     # NOTE: this function is generated by convert_hf_to_gguf_update.py
>     #       do not modify it manually!
>     # ref:  https://github.com/ggml-org/llama.cpp/pull/6920
>     # Marker: Start get_vocab_base_pre
> ```
> 
> I've never done ran that myself so I don't know how to use it but the style here looks inconsistent with other models, so makes me think it wasn't used?我从未亲自运行过那个，所以不知道如何使用它，但这里的风格与其他模型不一致，所以让我怀疑它没有被使用过？

No, I modified it based on GLM4, chatglm4, and glmedge, because the chkhsh of these models are all different.I haven’t used this script either.